### PR TITLE
Add Azure Container Registry conformance tests

### PR DESCRIPTION
**Pull request by gildardogmsft**
This PR adds the conformance results for Azure Container Registry

Signed-off-by: gildardogmsft gildardog@microsoft.com

### CODE REVIEW

**Comment by tianon** [comment]
@jdolitsky looking at the deploy for this, the TOC link has `#azurecontainerregistry` but the header it's supposed to link to has `id="azure-container-registry"` -- that mismatch doesn't seem intentional? :sweat_smile:

**Comment by tianon** [comment]
I guess to be more clear -- is that a bug in the generator code, or an issue that this PR needs to fix somehow?

File: distribution-spec/live/django-oci/badges.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[![Django OCI Testing](https://github.com/vsoch/django-oci/actions/workflows/main.yml/badge.svg)](https://github.com/vsoch/django-oci/actions/workflows/main.yml)
```
Comment (vsoch) [inline distribution-spec/live/django-oci/badges.md:1]: I think perhaps you are adding old commits (e.g., why is Django OCI being added here?)

**Comment by jdolitsky** [comment]
@tianon yea.. thats gonna be generator bug we should address

**Review by jdolitsky — CHANGES_REQUESTED** [review]
Appears to be several files modified unrelated to the change.. can you please rebase @gildardogmsft ?

**Review by vsoch — COMMENTED** [review]
@gildardogmsft really great that you are adding these!

It looks like you are bringing in previous commits (e.g., I'm seeing maintainer files and Django OCI) so I'd like to ask if you can please rebase with the main branch and push changes that only show updated files. Then we will be able to do a review proper.

File: distribution-spec/live/django-oci/badges.md (hunk 1 above)
Comment (gildardogmsft) [reply distribution-spec/live/django-oci/badges.md]: Sorry I did a wrong rebase :( I have rebased once again to only include the changes

**Comment by gildardogmsft** [comment]
> Appears to be several files modified unrelated to the change.. can you please rebase @gildardogmsft ?

Sorry I pushed a faulty rebase by mistake :( I have rebased again to only include the change

**Review by jdolitsky — APPROVED** [review]
This overall lgtm

File: distribution-spec/v1.0/azurecontainerregistry/PRODUCT.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+vendor: Microsoft
+name: Azure Container Registry
+version: 211214
+website_url: https://azure.microsoft.com/en-us/services/container-registry/
+documentation_url: https://docs.microsoft.com/en-us/azure/container-registry/
+product_logo_url: https://azure.microsoft.com/svghandler/container-registry/?width=600&height=315
+type: hosted
+description: Store images for all types of container deployments and OCI artifacts, using Azure Container Registry.
```
Comment (vsoch) [inline distribution-spec/v1.0/azurecontainerregistry/PRODUCT.yaml:8]:
```suggestion
description: Store images for all types of container deployments and OCI artifacts using Azure Container Registry.
```

**Review by tianon — APPROVED** [review]

File: distribution-spec/v1.0/azurecontainerregistry/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+export OCI_USERNAME="<username>"
+export OCI_PASSWORD="<password>"
+export OCI_NAMESPACE="conformance"
+export OCI_TEST_PULL=1
+export OCI_TEST_PUSH=1
+export OCI_TEST_CONTENT_DISCOVERY=1
+export OCI_TEST_CONTENT_MANAGEMENT=1
+```
+
+And finally run the conformance tests by running `./conformance.test`.
```
Comment (vsoch) [inline distribution-spec/v1.0/azurecontainerregistry/README.md:28]:
```suggestion
And finally, run the conformance tests by running `./conformance.test`.
```
[explanation](https://linguaholic.com/linguablog/comma-after-finally/#:~:text=%E2%80%9CFinally%E2%80%9D%20needs%20to%20be%20followed,comma%20if%20it%20is%20restrictive.)

**Comment by vsoch** [comment]
heeey I was reviewing! :cry:

**Review by vsoch — COMMENTED** [review]
I guess I'll submit anyway :P

**Comment by tianon** [comment]
Doh, sorry!  I'll go a little slower on the trigger next time :see_no_evil:

(It's also not too hard to revert or re-PR if something needs to change :sweat_smile:)

**Comment by vsoch** [comment]
Nah it's okay, they were tiny punctuation nit picks, not really worth that.

**Pull request merged**